subsys/bluetooth/controller/Kconfig

```diff
@@ -147,10 +151,17 @@ config BT_CTLR_HCI
 
 comment "BLE Controller configuration"
 
+config BT_CTLR_ENTROPY
+	bool "Random number generation in Controller"
+	depends on BT_CTLR_ENTROPY_SUPPORT
```

Comment:
Could you elaborate on why we cannot use `CONFIG_ENTROPY_HAS_DRIVER` or something from device tree here?